pyscf/df/hessian/rhf.py

```diff
@@ -94,17 +94,17 @@ def _partial_hess_ejk(hessobj, mo_energy=None, mo_coeff=None, mo_occ=None,
     int2c_ip1 = auxmol.intor('int2c2e_ip1', aosym='s1')
 
     rhoj0_P = 0
-    if with_k:
-        if hessobj.max_memory*.8e6/8 < naux*nocc*(nocc+nao):
-            raise RuntimeError('Memory not enough. You need to increase mol.max_memory')
-        rhok0_Pl_ = np.empty((naux,nao,nocc))
+    
+    if hessobj.max_memory*.8e6/8 < naux*nocc*(nocc+nao):
```

Comment:
This is sort of a funny line, no? `8/8`

Reply:
I guess it is just for better readability. 
0.8 is margin of safety. And 8 is the size of double type.